### PR TITLE
Add and fix aqua tests

### PR DESCRIPTION
**Pull request by josephmckinsey**
- Adds Aqua.jl tests
- Fixes an unbound arg problem in the helper_functions.jl
- Fixes missing compat for JSON and TimeSeries

### CODE REVIEW

File: src/util/sienna/helper_functions.jl [hunk 1]
```diff
@@ -151,9 +151,9 @@ function get_forecast_values(ts::Nothing)
 end
 
 function get_outage_time_series_data(
-    gen::Union{SI, BR},
+    gen::Union{PSY.StaticInjection, PSY.Branch},
```
Comment (scdhulipala) [inline src/util/sienna/helper_functions.jl:154]: I'm curious why this change? Is Aqua pointing to an ambiguity here?

Comment (scdhulipala) [reply src/util/sienna/helper_functions.jl:154]: Would this work on any subtype of PSY.Branch and PSY.StaticInjection?

**Review by scdhulipala — APPROVED** [review]

File: src/util/sienna/helper_functions.jl (hunk 1 above)
Comment (josephmckinsey) [reply src/util/sienna/helper_functions.jl:154]: Aqua is pointing out an unused type variable. Any subtype of PSY.Branch and PSY.StaticInjection should satisfy `T <: Union{PSY.StaticInjection, PSY.Branch}`
